File: src/Qwack.Transport/TransportObjects/Interpolators/TO_Interpolator1d.cs
using ProtoBuf;
using Qwack.Transport.BasicTypes;

namespace Qwack.Transport.TransportObjects.Interpolators
{
    [ProtoContract]
    public class TO_Interpolator1d
    {
        [ProtoMember(1)]
        public double[] Xs { get; set; }
        [ProtoMember(2)]
        public double[] Ys { get; set; }
        [ProtoMember(3)]
        public Interpolator1DType Type { get; set; }
        [ProtoMember(4)]
        public bool IsSorted { get; set; }
        [ProtoMember(5)]
        public bool NoCopy { get; set; }
    }
}

File: src/Qwack.Transport/GeneratedTypeScript/TO_Interpolator1d.ts

import { Interpolator1DType } from './Interpolator1DType';

export interface TO_Interpolator1d  {
	 xs: number[];
	 ys: number[];
	 type: Interpolator1DType;
	 isSorted: boolean;
	 noCopy: boolean;
}


